export const InternalErrorException =()=>{

    return <>
        <div>
            Exception happened...
            Try again....
        </div>
    </>
}

export default InternalErrorException;